llama/alma.py

```diff
@@ -121,17 +122,33 @@ def get_vendor_details(self, vendor_code):
         time.sleep(0.1)
         return r.json()
 
-    def mark_invoice_paid(self, invoice_id: str, invoice_xml_path: str) -> str:
+    def mark_invoice_paid(
+        self,
+        invoice_id: str,
+        payment_date: str,
```

Comment:
In general across this application we're passing around the datetime object and then creating the appropriate string formatting within the function where a string output is needed. I would suggest doing that here as well since it simplifies the processes calling this function and means if the format of the string ever needs to change, only the internals of this function will be impacted.